README.md

```diff
@@ -16,3 +16,47 @@ The project and Python dependencies can be installed from source with:
 ```
 pip3 install .
 ```
+
+## Configuration
```

Comment:
s/Configuration/Configuration System/

This section is not about configuring this tool,
but about the configuration framework that is provided.